Package.swift

```diff
@@ -11,13 +11,22 @@ let package = Package(
     .library(name: "PactConsumerSwift", targets: ["PactConsumerSwift"])
   ],
   dependencies: [
-    .package(url: "https://github.com/Quick/Nimble.git", from: "8.0.0")
+    .package(url: "https://github.com/Quick/Nimble.git", from: "8.0.0"),
+    .package(url: "https://github.com/Quick/Quick.git", from: "2.2.0")
```

Comment:
I guess there is no way to avoid this. Because it is tagged in the `testTarget` it will actually never end up in the client's dependencies. Same can be said for Nimble itself.

Reply:
Indeed SwiftPM will only include Nimble and Quick when running the unit tests for the PactConsumerSwift package.  When external projects bring in the PactConsumerSwift target they will not require Nimble or Quick.